src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -401,12 +403,12 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 		// update badge frame when a category is selected, unless no-hexagon-frame checkbox is checked
 		this.badgeClassForm.rawControl.controls['badge_category'].statusChanges.subscribe((res) => {
 			this.handleBadgeCategoryChange();
-			if(that.imageField.control.value){
+			if(that.imageField && that.imageField.control.value){
```

Comment:
You can use [optional chaining](https://blog.logrocket.com/optional-chaining-nullish-coalescing-typescript/#optional-chaining) to make this more readable:
```ts
			if(that.imageField?.control.value){
```

Reply:
updated